internal/providers/terraform/aws/registry.go

```diff
@@ -302,6 +303,7 @@ var FreeResources = []string{
 	"aws_db_instance_role_association",
 	"aws_db_parameter_group",
 	"aws_db_subnet_group",
+	"aws_db_option_group",
```

Comment:
We try to keep the list in alphabetical order for easier search. Could you please move this line above `aws_db_parameter_group`?